BigBaseV2/src/util/vehicle.hpp

```diff
@@ -75,6 +107,10 @@ namespace big::vehicle
 			Vehicle veh = VEHICLE::CREATE_VEHICLE(hash, location.x, location.y, location.z, heading, is_networked, false, false);
 			*(unsigned short*)g_pointers->m_model_spawn_bypass = 0x0574;
 
+			if (plate[0] != 0) {
+				VEHICLE::SET_VEHICLE_NUMBER_PLATE_TEXT(veh, plate);
+			}
+
```

Comment:
We would like to keep the spawn function clear from modifications, this way we spawn a clean vehicle on which we can put the modifications. Suggested to move the plate to a separate function in the vehicle util.

Reply:
Moved it to set_plate() function